containers/Loans/Loans.tsx

```diff
@@ -71,6 +72,21 @@ function Container() {
 		};
 	}, [erc20LoanContract, ethLoanContract, isL2]);
 
+	const subgraphOpenLoansQuery = subgraph.useGetLoans(
+		{ where: { isOpen: true, account: address } },
+		{ id: true, collateralMinted: true },
+		{ queryKey: ['getLoans', isL2, address] }
+	);
+
+	const subgraphOpenLoansKey = subgraphOpenLoansQuery.data
+		? JSON.stringify(
+				subgraphOpenLoansQuery.data.map((x) => {
+					const id = x.id.replace(/-\w+/, ''); //remove -sETH from id
+					return { ...x, id };
+				})
+		  )
+		: '';
```

Comment:
I don't understand the reason behind stringifing the array and then parsing it down in the code

Reply:
We want to rerun this useEffect when there is a change in openLoans. You cant put and array in the `useEffect`s dependency array

Reply:
Nice, didn't notice that